libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -467,7 +468,6 @@ bool ASTJsonConverter::visit(UserDefinedTypeName const& _node)
 	setJsonNode(_node, "UserDefinedTypeName", {
 		make_pair("name", namePathToString(_node.namePath())),
 		make_pair("referencedDeclaration", idOrNull(_node.annotation().referencedDeclaration)),
-		make_pair("contractScope", idOrNull(_node.annotation().contractScope)),
```

Comment:
Why is this removed? Is this a breaking change?

Reply:
It is always null because `contractScope` is never assigned.